<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="PTE_Input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="PTE_Console">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_PageNo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_PrevPage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev Page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_NextPage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Update">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>290</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Update</string>
    </property>
   </widget>
   <widget class="HNS_Message_Preview" name="GV_Preview" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>681</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Send">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CMB_Serial">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Connect">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Disconnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QSlider" name="SLD_Time">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>700</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>680</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Time">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>720</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="HNS_Signboard_Preview" name="GV_Preview_2" native="true">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>681</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Create_Sign">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Signboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_FontTest">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Count</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Signboard_Preview</class>
   <extends>QWidget</extends>
   <header>hns_signboard_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
